common/containers/Tabs/SendTransaction/index.jsx

```diff
@@ -238,8 +267,63 @@ export class SendTransaction extends React.Component {
     return { to, data, value, unit, gasLimit, readOnly };
   }
 
+  isValid() {
+    const { to, value } = this.state;
+    return (
+      isValidETHAddress(to) &&
+      value &&
+      Number(value) > 0 &&
+      !isNaN(Number(value)) &&
+      isFinite(Number(value))
+    );
+  }
+
+  // FIXME MOVE ME
+  getTransactionFromState() {
+    // FIXME add gas price
+    if (this.state.unit === 'ether') {
+      return {
+        to: this.state.to,
+        from: this.props.wallet.getAddress(),
+        // gasPrice: `0x${new Number(50 * 1000000000).toString(16)}`,
+        value: '0x' + toHex(toWei(Big(this.state.value), 'ether'))
+      };
+    }
+    const token = this.props.tokens.find(x => x.symbol === this.state.unit);
+    if (!token) {
+      return;
+    }
+
+    return {
+      to: token.address,
+      from: this.props.wallet.getAddress(),
+      // gasPrice: `0x${new Number(50 * 1000000000).toString(16)}`,
```

Comment:
In #100 I have to go to the node to fetch the gasPrice value, so presumably that's what needs to happen. I'll just leave that work in there and reconcile the two, with whichever we merge first.

https://github.com/MyEtherWallet/MyEtherWallet/pull/100/files#diff-caeda19c2aed697b1267cd27ba1f9844R86